### PR TITLE
Strip mscorlib.xml from mscorlib.dll when linking

### PR DESCRIPTION
**Pull request by chamons**
- Refactor skip logic into function for readability

### CODE REVIEW

File: tuner/Mono.Tuner/RemoveResources.cs [hunk 1]
```diff
@@ -33,19 +33,27 @@ public virtual void Process (LinkContext context)
 				if (resource == null)
 					continue;
 
-				switch (resource.Name) {
+				if (!SkipResource (resource.Name)) {
```
Comment (marek-safar) [inline tuner/Mono.Tuner/RemoveResources.cs]: The logic seems to be inverse you skip resource on `!SkipResource` ?

Comment (chamons) [reply tuner/Mono.Tuner/RemoveResources.cs]: You are right, the name is wrong. I had inverted it and did not go back to fix it.

Good catch.

**Comment by chamons** [comment]
Also just realized my formatting was wrong. Fixing that as well.

**Comment by marek-safar** [comment]
build

**Comment by chamons** [comment]
I _believe_ I got the formatting right this type. The switch formatting still looks wrong to my eyes even after multiple years.

**Pull request merged**